docs/tags/global.md

```diff
@@ -141,6 +141,37 @@ Holds up processing. This does not schedule execution for later, it pauses execu
 
 Evaluates math. `precision_fix` enables a hack that fixes floating point precision errors, but may cause issues in very specific circumstances. Realistically nothing you do with actions should require disabling `precision_fix`.
 
+
+```json
+// Supported operators
+{math; 1 + 1} // Addition
+{math; 1 - 1} // Substraction
+{math; 1 * 1} // Multiplication
+{math; 1 / 1} // Division
+{math; 1 ^ 1} // Power
+{math; 1 % 1} // Modulo
+
+// Bitwise operators
+{math; 0011 | 0101} // Logical OR (0111)
+{math; 0011 & 0101} // Logical AND (0001)
+
+// Supported functions
+{math; sqrt(1)} // Square-root
+{math; log(1)}  // Logarithm
+
+// Trigonometry functions. All of them are using values in degrees.
+{math; sin(1)} // Sine
+{math; cos(1)} // Cosine
+{math; tan(1)} // Tangent
+```
+
+### Use-case
```

Comment:
I'd move this into the examples block and just stick it at the end with 
```
// Usage
{...}
```
every other tag uses the first block as the examples of how to use it

Reply:
I agree